docs/source/contribution/technical_steering_committee.md

```diff
@@ -48,40 +48,41 @@ We look for commitment markers who can do the following:
 - Show excitement about the future of Kedro
 - Build a collaborative relationship with the existing team
 
-## Kedro maintainers
+## Current maintainers
 
 <!-- DO NOT EDIT THIS AND MERGE A PR WITHOUT A VOTE TO SIGN OFF ANY CHANGES -->
 
-Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) and [Nikolaos Tsaousis](https://github.com/tsanikgr) at [QuantumBlack](https://www.mckinsey.com/capabilities/quantumblack) to solve challenges they faced in their project work. Their work was later turned into an internal product by [Peteris Erins](https://github.com/Pet3ris), [Ivan Danov](https://github.com/idanov), [Nikolaos Kaltsas](https://github.com/nikos-kal), [Meisam Emamjome](https://github.com/misamae) and [Nikolaos Tsaousis](https://github.com/tsanikgr).
-
+| Name                                                     | Organisation |
+|----------------------------------------------------------|------------- |
+| [Ahdra Merali](https://github.com/AhdraMeraliQB)         | QuantumBlack |
+| [Andrew Mackay](https://github.com/Mackay031)            | QuantumBlack |
+| [Ankita Katiyar](https://github.com/ankatiyar)           | QuantumBlack |
+| [Antony Milne](https://github.com/antonymilne)           | QuantumBlack |
+| [Deepyaman Datta](https://github.com/deepyaman)          | Claypot AI   |
+| [Dmitry Sorokin](https://github.com/DimedS)              | QuantumBlack |
+| [Huong Nguyen](https://github.com/Huongg)                | QuantumBlack |
+| [Ivan Danov](https://github.com/idanov)                  | QuantumBlack |
+| [Jitendra Gundaniya](https://github.com/jitu5)           | QuantumBlack |
+| [Jo Stichbury](https://github.com/stichbury)             | QuantumBlack |
+| [Joel Schwarzmann](https://github.com/datajoely)         | QuantumBlack |
+| [Juan Luis Cano](https://github.com/astrojuanlu)         | QuantumBlack |
```

Comment:
@astrojuanlu @stichbury not sure if you prefer "QuantumBlack Labs", "QuantumBlack, a McKinsey company", or something else (for everybody).

Reply:
QuantumBlack, AI by McKinsey with a link to https://www.mckinsey.com/capabilities/quantumblack pls.

Reply:
Will do! For consistency, I will do GetInData | Part of Xebia and link that (as well as linking Claypot AI and Societe Generale with the correct accents).